enhancements/windows-containers/container-runtime-containerd.md

```diff
@@ -0,0 +1,213 @@
+VMs that will be created with containerd as default runtime. All minimum requirements stated
+as part of WMCO upgrade will be applicable here as well. The golden image that we used to create
+Windows VM doesn't need to have docker runtime. It is optional as customers may use docker for
+non-kubernetes requirements.
+2) BYOH ( Bring Your Own Host)
+  If Windows VM was created by BYOH option, the upgrade process will uninstall all components
+that were installed by WMCO. As part of the de-configuring process, Windows OS specific configurations
+like HNS-Network that were created by WMCO would have been deleted. During reinstall workflow, containerd
+will be installed along with kubelet, kube-proxy, CNI and hybrid-overlay.
+
+There is no difference between MachineSet and BYOH on how we enable containerd as default runtime. 
+
+Containerd Migration plan
+   *Containerd will become default runtime as part of OpenShift 4.11
+   *For current usage, we plan to introduce WMCO feature flag and release it only for community operator.
+   *The default value will be Docker CRI. In 4.10 community branch, the default value will be containerd.
+   *Customers will not have the option to choose runtime by using WMCO feature flag.
+   *In BYOH case Windows VM Will be de-configured and re-configured with the WMCO supported runtime.
+   *In MachineSet option Docker CRI based VM will be deleted and a new VM will be created with containerd
+    as default runtime.
+
+### User Stories
+
+Stories can be found within the [Windows Containers: containerd epic](https://issues.redhat.com/browse/WINC-505)
+
+### Justification
+
+*If we don't make containerd as default runtime, we will be left out with Mirantis supported dockershim
+and will have to re-design WMCO to incorporate Mirantis dockershim.
+*From the business point of view subscribing to Mirantis for dockershim support is not an ideal situation as Mirantis
+ is our key competitor.
+*If we want to use CRI-O for Windows, the amount of time and engineering efforts involved making CRI-O to work for 
+Windows is huge. This doesn't do business justification due to the high cost and time. 
+*Containerd is widely adopted and supported by the open source community. 
+*Microsoft is a major contributor in Windows containerd development and moved to containerd as default runtime for their 
+kubernetes offerings. 
+*Most of the Windows-supported kubernetes orchestrators already moved to containerd.
+*Containerd has become the default runtime in linux/windows nodes.
+*Testing and adaptation will require less engineering effort compare to other solutions due to wider open source
+ community and enterprise support.
+
+### Design Details
+
+we plan to use containerd 1.6.0 to integrate into WMCO. Containerd will be installed as a first service before 
+kubelet installation as kubelet must require containerd to be running. We already bundled containerd as part of 
```

Comment:
`...kubelet requires containerd to be running...`

You can leave out `We already bundled...` but rather state that containerd will be bundled as part of WMCO in the initial statement.